packages/block-library/src/latest-posts/edit.js

```diff
@@ -303,21 +303,35 @@ export default function LatestPostsEdit( { attributes, setAttributes } ) {
 								} )
 							}
 						/>
-						<BaseControl className="editor-latest-posts-image-alignment-control">
-							<BaseControl.VisualLabel>
-								{ __( 'Image alignment' ) }
-							</BaseControl.VisualLabel>
-							<BlockAlignmentToolbar
-								value={ featuredImageAlign }
-								onChange={ ( value ) =>
-									setAttributes( {
-										featuredImageAlign: value,
-									} )
-								}
-								controls={ [ 'left', 'center', 'right' ] }
-								isCollapsed={ false }
+						<ToggleGroupControl
+							className="editor-latest-posts-image-alignment-control"
```

Comment:
I guess we should leave this class for back compat, but we can remove the associated CSS in `editor.scss`.